src/script/components/MessagesList/Message/ContentMessage/asset/VideoAsset.tsx

```diff
@@ -77,12 +82,39 @@ const VideoAsset: React.FC<VideoAssetProps> = ({
     if (assetPreviewResource && isFileSharingReceivingEnabled) {
       getAssetUrl(assetPreviewResource).then(setVideoPreview);
     }
+
     return () => {
       videoPreview?.dispose();
       videoSrc?.dispose();
     };
   }, []);
 
+  // Initial check if video is supported with `canPlayType` method, which checks for MIME type, e.g. 'video/mp4' or 'video/mov'.
+  // It's not 100% reliable (e.g. doesn't check codecs), but it's synchorous, which is helpful for initial rendering.
+  const isVideoMimeTypeSupported = (mimeType: string): boolean => {
+    const video = document.createElement('video');
+    const canPlay = video.canPlayType(mimeType) !== '';
```

Comment:
Just a suggestion, if we create an enum for the return types it would be more readable:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType#return_value

Reply:
I don't think that's necessary because we don't use it directly - we just check that it's not an empty string.